README.md

```diff
@@ -1,3 +1,5 @@
+[![Crash Free Sessions Rate](https://img.shields.io/badge/Crash_Free_Session_Rate-91.631%25-blue)](https://a8c.sentry.io/projects/studio/?project=4506612776501248&statsPeriod=7d)
```

Comment:
Shouldn't it be 93.623% as for now? What value do we exactly use?

Reply:
[The value I'm using is the one for the last seven days for all versions.](https://a8c.sentry.io/projects/studio/?project=4506612776501248&statsPeriod=7d) I.e. the same shown when clicking on the badge. It's true it changed since when I created the PR, the latest value is: 92.02%. I'll update it asap.

Alternatively, we could us the rate from the latest version. WDYT?

Reply:
It's okay as it is now, thanks.